app/components/project-icon.cjsx

```diff
@@ -24,20 +24,23 @@ module.exports = React.createClass
       @getDetails nextProps.project
 
   getDetails: (project) ->
-    apiClient.type 'avatars'
-      .get project.links.avatar.id
-      .then (avatar) =>
-        @setState {avatar}
-      .catch =>
-        project.get 'avatar'
-          .catch =>
-            null
-          .then (avatar) =>
-            @setState {avatar}
+    if project.avatar_src
+      @setState avatar: "http://#{ project.avatar_src }"
+    else
+      apiClient.type 'avatars'
+        .get project.links?.avatar?.id
+        .then (avatar) =>
+          @setState avatar: avatar.src
+        .catch =>
+          project.get 'avatar'
+            .catch =>
+              null
+            .then (avatar) =>
+              @setState avatar: avatar.src
 
```

Comment:
this block was a horrible hack on my part to get avatars working again, after the stats page broke. It would be better if we could retrieve avatars without making two API calls per project. I think the avatar should already be present in the resource if we're using the cards route, so maybe we don't need the `else` block?